src/components/form/Select.jsx

```diff
@@ -0,0 +1,113 @@
+import PropTypes from "prop-types";
+import { Listbox, Transition } from "@headlessui/react";
+import theme from "@/themes/theme";
+
+const Select = ({ label, value, onChange, list }) => (
+  <div css={[theme.select.wrapper]}>
+    <Listbox
+      as="div"
+      css={[theme.select.select]}
+      value={value}
+      onChange={onChange}
+    >
+      {({ open }) => (
+        <>
+          <Listbox.Label id="listbox-label" css={[theme.select.label]}>
+            {label}
+          </Listbox.Label>
+          <div css={[theme.select.main]}>
+            <span css={[theme.select.button.wrapper]}>
+              <Listbox.Button css={[theme.select.button.base]}>
+                <span css={[theme.select.button.value]}>{value}</span>
+                <span css={[theme.select.button.iconWrapper]}>
+                  <svg
+                    css={[theme.select.button.icon]}
+                    viewBox="0 0 20 20"
+                    fill="none"
+                    stroke="currentColor"
+                  >
+                    <path
+                      d="M7 7l3-3 3 3m0 6l-3 3-3-3"
+                      strokeWidth="1.5"
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                    />
+                  </svg>
+                </span>
+              </Listbox.Button>
```

Comment:
Please create a `SelectButton` component using the `Button` component as a foundation.  Then import that `SelectButton` component into this file [in other words, we are using a similar pattern that we used with `Input.jsx`